fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1998,6 +1998,12 @@ public class Config extends ConfigBase {
     @ConfField(masterOnly = true)
     public static int hms_events_polling_interval_ms = 10000;
 
+    /**
+     * Regex pattern of HMS databases synchronization, empty string means synchronizing all databases.
+     */
+    @ConfField(mutable = true, masterOnly = true)
+    public static String hms_sync_database_pattern = "";
```

Comment:
How about making it as a property of a certain Catalog?
So that we can specify different pattern for different Catalog

Reply:
@morningman This config is only for `HMSExternalCatalog`, maybe it's already a property of a certain Catalog. Can you further clarify your point for me?